repos/system_upgrade/el7toel8/actors/openldapnsstoopenssl/libraries/sssd.py

```diff
@@ -0,0 +1,139 @@
+# vim: tabstop=8 expandtab shiftwidth=4 softtabstop=4
+import os.path as ph
+
+from utils import (
+    NotNSSConfiguration,
+    InsufficientTooling
+)
+from openldap import Openldap
```

Comment:
```suggestion
from leapp.libraries.actor.openldap import Openldap
```

Reply:
Thanks for these which made me retry this approach which used to fail for me some time ago.